rep-0147.rst

```diff
@@ -187,31 +187,33 @@ The reference position is the origin of the ``base_link`` as defined in [#REP105
 Rate Interface
 ,,,,,,,,,,,,,,
 
-The messge to send a velocity command is ``geometry_msgs/Twist`` [#twist]_.
+The messge to send a velocity command is ``geometry_msgs/TwistStamped`` [#twiststamped]_.
 This is the standard way to command ROS vehicles.
```

Comment:
These two statements are no longer valid. ROS vehicles are typically commanded with `Twist`. This datatype _is_ stamped.